src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java

```diff
@@ -0,0 +1,176 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+
+public class ColumnViewUIFormatterDTO {
+
+    String columnName;
+    String dataType;
+    Integer regexId;
+    String regexGroup;
+    String regexLabel;
+    Boolean isSensitive;
+    Boolean isProtected;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocVariableCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java L13
    
    Missing a Javadoc comment.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1922538048>*